server/services/notify/plugindelivery/plugin_delivery.go

```diff
@@ -80,6 +81,18 @@ func (pd *PluginDelivery) Deliver(mentionUsername string, extract string, evt no
 		ChannelId: channel.Id,
 		Message:   formatMessage(author.Username, extract, evt.Card.Title, link, evt.BlockChanged),
 	}
+
+	embed := cardEmbed{
+		OriginalPath: link,
```

Comment:
This will cause issues becuase this was intended to be `u.RequestURI(),` 

Input
```Go
package main
 
import (
   "fmt"
   "net/url"
)
 
func main() {
 
   str := "http://ispycode.com/GO/URL/Get-Request-URI"
   u, _ := url.Parse(str)
   uri := u.RequestURI()
   fmt.Println("uri:", uri)
}
```

Output
```Go
$ go run example.go
uri: /GO/URL/Get-Request-URI
```

We don't want the full link, just the path.

Reply:
Ok, will fix.

Reply:
... strangely my tests still worked with the full URL.

Reply:
It's for frontend, not backend. This has no effect on the backend (when you click on the preview to take you to the card)

Reply:
Fixed